jenkins-design-language/src/js/components/CommitId.jsx

```diff
@@ -9,7 +9,7 @@ const GIT_HASH_REGEX = /\b[0-9a-f]{5,40}\b/;
  */
 export class CommitId extends Component {
     render() {
-        const commitId = this.props.commitId;
+        const {commitId, url, title} = this.props;
```

Comment:
if commitId is null - still ok with destructuring like this? (my js foo is weak)